content/post/golang-api.md

```diff
@@ -0,0 +1,317 @@
++++
+title= "Go Go, Go! Stream Processing for Go"
+date = 2018-01-18T12:00:00-04:00
+draft = false
+author = "aturley"
+description = "Wallaroo’s Go API opens the door to writing scale-independent applications in Go."
+tags = [
+    "wallaroo",
+    "golang",
+    "api"
+]
+categories = [
+    "Go API"
+]
++++
+
+We've been working on our processing engine, [Wallaroo](https://github.com/wallaroolabs/wallaroo/tree/release) for just under two years now. Our goal has been to make it as easy to build fast, scale-independent applications for processing data. When we open sourced Wallaroo last year we provided an API that let developers create applications using [Python](python-api.md). Over the last few months we've been working to make Wallaroo available to a wider range of programmers by adding a Go API, and that's what I'd like to talk about today.
```

Comment:
Python link is broken.

Should we link to `release` or `master` for Wallaroo? I believe previously we were linking to `master`.